skbio/diversity/beta/tests/test_unifrac_base.py

```diff
@@ -0,0 +1,612 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+from io import StringIO
+
+import numpy as np
+
+from skbio import TreeNode
+from skbio.tree import DuplicateNodeError, MissingNodeError
+
+
+class StatsTests(object):
+    _method = {}
+
+    def setUp(self):
+        self.unweighted_unifrac = self._method['unweighted_unifrac']
```

Comment:
Great, thanks, and no other changes. It is likely this strategy can be used generally when we have multiple implementations of methods.